gcp/appengine/blog/content/posts/using-the-determineversion-api.md

```diff
@@ -0,0 +1,180 @@
+---
+title: "Using the determineversion API to find C/C++ vulnerabilities"
+date: 2023-07-18T19:16:34Z
+draft: true
+author: OSV Team
+---
+With the increasing incidence of software supply chain attacks, it is more important than ever for developers to understand the known vulnerabilities in their open source dependencies, regardless of the ecosystem of origin. Within the C/C++ ecosystem it is difficult to match dependencies to vulnerabilities for a few of reasons:
+
+- C/C++ does not have a centralized package manager like npm or pyPI
+- Software projects typically pull in C/C++ by submodules or vendoring
+- Source code identifiers (e.g. git hashes) are the best way to identify libraries, but vulnerabilities are typically associated to versions, not git hashes
+
+OSV has had C/C++ vulnerability data from OSS-Fuzz keyed on git hashes from day 1. However, a remaining challenge for C/C++ users is being able to accurately identify the closest upstream git hash of their C/C++ dependencies in order to make use of this vulnerability data. The OSV team is committed to bridging the gap between what C/C++ users need and the constraints of the ecosystem and today we are announcing a tool that can help–the determineversion API. 
+<!--more-->
+
+## What is the determineversion API?
+The [determineversion API](https://google.github.io/osv.dev/post-v1-determineversion/) is a new, experimental API endpoint for OSV’s API. The goal of the determineversion API is to help users determine the likely version of their vendored C/C++ dependencies. Once dependency versions are known, our other API endpoints can return reliable vulnerability information. 
+
+We are actively seeking feedback on the determineversion API. If you have a project that you would like to scan for vulnerabilities, please follow along with this walkthrough and let us [know about your experience](https://github.com/google/osv.dev/issues/new). Your feedback can help us improve the vulnerability matching experience for other C/C++ developers. 
```

Comment:
This paragraph seems out of place here. Would it work better at the end?

Reply:
Moved.